core/domain/summary_services.py

```diff
@@ -114,7 +114,7 @@ def get_learner_collection_dict_by_id(
         completed_exp_ids = []
 
     collection_dict = collection.to_dict()
-    collection_dict['skills'] = collection.skills
+    collection_dict['skills'] = collection.get_human_readable_skills()
```

Comment:
so should we change the key to skill_names or human_readable_skill_names or something like that, to avoid confusion with the regular 'skills' attribute?